sorc/ncep_post.fd/PARA_RANGE.f

```diff
@@ -46,4 +46,42 @@ SUBROUTINE PARA_RANGE (N1,N2,NPROCS,IRANK,ISTA,IEND)
       if ( iwork2 > irank ) iend = iend + 1
       return
       end
+!!
+!! USAGE:    CALL PARA_RANGE2(N1,N2,NX,NY,NRANK,ISTA,IEND,JSTA,JEND)(A)
+!!   INPUT ARGUMENT LIST:
+!!     N1 - LAAT INTERATE VALUE I dimension
+!!     N2 - LAST INTERATE VALUE  J dimension
+!!     NX  NUMBER OF subdomains in Z  dimension
```

Comment:
in X dimension

Reply:
@GeorgeVandenberghe-NOAA Checked the updated version of para_range.f, mpi_first.f and ctlblk.f and they look fine.